runtime/util/finish_rewriting.cpp

```diff
@@ -15,7 +15,7 @@ bool statistics = false;
 bool binary_output = false;
 bool proof_output = false;
 
-extern int64_t steps;
+int64_t get_steps();
```

Comment:
Not sure why this was a signed integer before, but I think it should an unsigned integer, ie `uint64_t`